personalization.tracker.inc

```diff
@@ -327,6 +327,7 @@ function personalization_get_user() {
     }
     else {
       $uid = uniqid(NULL, TRUE);
+      $_COOKIE['pz_user_id'] = $uid;
```

Comment:
Not from this PR but I think we have a bit of a security hole with this cookie. We use it's value to write and update data in the database but there's nothing stopping you changing your cookie and affecting someone elses content.

For example if there was a page on your site about my product, I could influence peoples personalisations score by pretending to be them so my content is surfaced.